server/src/integrations/message-queue/drivers/local.driver.ts

```diff
@@ -0,0 +1,20 @@
+import { EventEmitter } from 'stream';
+
+import { MessageQueueDriver } from './interfaces/message-queue-driver.interface';
+
+export class LocalDriver extends EventEmitter implements MessageQueueDriver {
+  constructor() {
+    super();
+  }
+
+  async subscribe<T>(
```

Comment:
How do we use this local driver in practise? In Laravel if I remember well, that driver was called "sync" and the jobs where process synchronously if you had not configured any driver. Maybe we should do something similar and subscribe to it when it's that driver? Otherwise here the jobs will just get lost, no?

Reply:
not really, this local driver works like any async queue but it uses nodejs built in event emiter(no extra dependency), its only meant for local usage only and it provides the same publish subscribe interface, so that users of queue dont have to worry about which driver is configurred, and none of the jobs gets lost here.

Reply:
Yes sorry I wasn't clear. I meant that we lack something to start listening/processing the queue so we'd be better off converting this to a synchronous driver so at least it's useful. But actually I realize the previous comment also apply to pg-boss so my suggestion was a bit irrelevant, it's just that we don't really have a standard script to let the user start a worker at this point (or I missed it). Maybe add a yarn/npm command?

Reply:
so i have not added workers yet, its just queue, but to use the localdriver no extra setup will be required, for pg-boss just `yarn` command is sufficient which is run as part of setup anyway so, in practice users dont have to do any setup, even local driver is irrelavant as pg-boss uses postgress and users needs to have postgress for 20 to work. so i dont think there will be any problems or challenges while setting up.

Reply:
Yes true. Maybe the synchronous driver could be useful to debug more easily at least!
I meant creating worker file/command that starts and processes the queue for any driver. Good if it's planned

Reply:
yup, i will add 1st worker module during the notifcation implementation